src/pages/home/report/ReportAttachments.js

```diff
@@ -38,7 +39,11 @@ function ReportAttachments(props) {
             defaultOpen
             report={report}
             source={source}
-            onModalHide={() => Navigation.dismissModal()}
+            onModalHide={() => {
+                Navigation.dismissModal();
+                // This enables Composer refocus when the attachments modal is closed by the browser navigation
+                ComposerFocusManager.setReadyToFocus();
```

Comment:
@abdulrahuman5196 I decided to go with my alternative solution for the Composer refocus on navigation because removing the `!fullscreen` check in `hideModal` would cause a regression on the Composer refocus after closing the Emoji picker.

Reply:
Do we need this change? What does change bring in? @paultsimura

Reply:
This focuses the composer when we navigate back by clicking the browser's "Back" button. Otherwise, `ComposerFocusManager.setReadyToFocus()` is called within `BaseModal` in 2 places: 

https://github.com/Expensify/App/blob/d521dd6783c382297e06a429701ae1936f808041/src/components/Modal/BaseModal.tsx#L115-L117

https://github.com/Expensify/App/blob/d521dd6783c382297e06a429701ae1936f808041/src/components/Modal/BaseModal.tsx#L66-L68

The `AttachmentsModal` is fullscreen, so the `hideModal` does not call the `ComposerFocusManager.setReadyToFocus()`.
Also, the Modal's `onDismiss` is not called when we are navigating – it's the RN limitation. The component just gets destroyed without calling the `onDismiss`.
Therefore, the `ComposerFocusManager.setReadyToFocus()` is not called in these two places in the `BaseModal`, so we need to call it explicitly.